src/linker/Linker.Steps/SweepStep.cs

```diff
@@ -97,14 +97,14 @@ void UpdateAssemblyReferencesToRemovedAssemblies (AssemblyDefinition assembly)
 		{
 			var action = Annotations.GetAction (assembly);
 			switch (action) {
-			case AssemblyAction.Skip:
+			case AssemblyAction.Copy:
+			case AssemblyAction.CopyUsed:
```

Comment:
I know we discussed the idea of changing `CopyUsed` to be like `Copy` when it's marked, but I don't remember if we decided to do it. As-is I think it has the same problem as `Copy` where static typerefs won't get updated, but if we fix that it could bring in more facades than we want. I think it would be safest to keep the current behavior for back-compat, and just recommend not using `CopyUsed`.